### DISCUSSION TITLE
Trying to use this.wdioRetries, but always get it's equal to 0

### DISCUSSION THREAD

**Discussion by opallavihb** [webdriverio/webdriverio#11911] [Q&A]
my code snip: 

```
let utils = require("../utils/utils");
const { default: newFlowScreen } = require("../../pageobjects/flow/general/createFlowScreen");
const { default: mainMenuScreen } = require("../../pageobjects/general/mainMenuScreen");
const { testData } = require("../../../testData");
const commonActions = require("../process/commonActions");

describe("Retry trial test", () => {
	it("Login to application", async function() {
             console.log('Retry count:', this.wdioRetries);
             await commonActions.loginToApplication(testData.retryTrialTest.user, testData.retryTrialTest.pass);
	});

	it("Enter the project", async function () {
             await expect(await mainMenuScreen.getProject(testData.retryTrialTest.projectName)).toExist({
		   	message: `Project name ${testData.retryTrialTest.projectName} not found.`,
	     });
	    await commonActions.enterProject(testData.retryTrialTest.projectName);
            await commonActions.moveToQuestionnarieIfNeeded();
	});

	it("Empty Screen",async function () {
	    // await commonActions.createButton();
	    await newFlowScreen.clickOnStartFromBlank();
	});
});
```

**the code is failing and being retried in my console - but my console.lof says that my Retry count: 0**

in my config:

```
  specFileRetries: 1,
  mochaOpts: {
    ui: "bdd",
    timeout: 600000,
    retries: 1,
    bail: false
  },
```

my webdriverIO version in my package.json is: "webdriverio": "^8.26.3"
what am I doing wrong?

Thanks in advance!

**Comment by christian-bromann** [comment]
@opallavihb thanks for raising the issue. Mind providing a minimal reproducible example?

**Reply by opallavihb** [reply]
Well all the info I added is what I can give :sweat_smile:
hope it's enough to help me!

**Reply by christian-bromann** [reply]
Unfortunately I can't do further investigations then. No one will spend time trying to reproduce a problem that you experience at the risk that it might be related to your environment. That is why it is important to provide a reproducible example ideally using a new project that highlights the problem and allows maintainers to experience it themselves very easy.

**Reply by opallavihb** [reply]
I don't understand what information you would like like me to add? maybe I didn't get it right

**Reply by BorisOsipov** [reply]
Shortly: 

Open https://webdriver.io/docs/gettingstarted
Create a brand new project via wizard.
Remove the auto-generated specs code.
Add your code snippets to the project to reproduce the problem.
Run your code and check the problem still appears.
Push this project to github.
Share the project link here.

It is unlikely that someone will do this for you to repeat your problem. Please make it as easy as possible to reproduce your problem. This will allow more people to help you.

**Reply by opallavihb** [reply]
Hey! I tried to create a new project but I'm getting lots of issues with that.
Is there a chance to please schedule a very small meeting with one of your support and I will show you my issue? It will help me a lot!!

**Reply by opallavihb** [reply]
Is there a chance maybe? :pray::skin-tone-3:
